src/main/java/facade/UserFacade.java

```diff
@@ -11,13 +13,37 @@
 public class UserFacade {
 
   private static final Logger LOGGER = LoggerFactory.getLogger(UserFacade.class);
+  private UserDAO userDao;
+
+  public UserFacade(){
+      userDao = new UserDAO();
+  }
+
+  private boolean validateUserCookie(String cookie){
+      //TODO: check cookie for valid user
+      String cookieUserName = null;
+      String cookiePassword = null;
+      User tempUser = null;
+      try {
+          tempUser = userDao.getUser(cookieUserName);
```

Comment:
To validate a cookie we may need to have some sort of algorithm that takes the username and password of the user and hashes them.